src/locales/de.json

```diff
@@ -124,5 +124,11 @@
   "errorText": "Upsi, es ist ein Fehler aufgetreten. Bitte versuche es erneut.",
 
   "export": "Exportieren",
-  "Logout": "Logout"
+  "Logout": "Logout",
+  "Save": "Speichern",
+  "No, cancel": "Nein, abbrechen",
```

Comment:
Maybe we just use cancel here or "Abbrechen"?

Reply:
Actually it's no exactly cancel, but more a oh i don't wanna give my email, so lets back